fog/ledger/connection/src/key_image.rs

```diff
@@ -59,9 +61,10 @@ impl FogKeyImageGrpcClient {
         };
 
         let response: CheckKeyImagesResponse =
-            retry(Fixed::from_millis(100).take(5).map(jitter), || {
+            retry(self.grpc_retry_config.get_retry_iterator(), || {
```

Comment:
How about we add a helper that forwards the call to `retry`?

Cals would then look like:
```rust
self.grpc_retry_config.retry(|| { ... })
```

Reply:
That's a good idea, will do